bundles/org.openhab.binding.airgradient/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="airgradient" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>AirGradient Binding</name>
+	<description>This is the binding for AirGradient.</description>
+
```

Comment:
Can you add some additional tags as connection, country's if possible?

Reply:
Valid for all countries, but added hybrid connection and autodiscovery.

I could not find any autodiscovery for "things on the device" That might be something to consider adding to the XML schema. E.g. if I connect to a hue hub, I find many lightbulbs. The hub is "local" - but what about the bulbs?